src/deriveAddress.c

```diff
@@ -15,7 +15,7 @@
 
 #define MAX_BIP32_PATH 32
 
-static void io_exchange_address();
+static void io_respond_with_address(uint8_t* addressBuffer, size_t addressSize);
 void ui_idle();
 
 derive_address_data_t daData;
```

Comment:
Could you please add a todo to move this into global state? (also "daData" sounds like a bad Russian name :-) (I know it is derive address data but ...)